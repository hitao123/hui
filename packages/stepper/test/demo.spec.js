describe('stepper component', () => {
  it('should show stepper', () => {

  });
});
