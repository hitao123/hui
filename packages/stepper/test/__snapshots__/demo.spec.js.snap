// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div class="demo-stepper">
    <div class="h-stepper"><button class="h-stepper__minus"></button> <input type="number" value="5" class="h-stepper__input"> <button class="h-stepper__plus"></button></div>
  </div>
  <div class="demo-stepper">
    <div class="h-stepper"><button class="h-stepper__minus h-stepper__minus--disabled"></button> <input type="number" value="1" class="h-stepper__input"> <button class="h-stepper__plus"></button></div>
  </div>
  <div class="demo-stepper">
    <div class="h-stepper"><button class="h-stepper__minus h-stepper__minus--disabled"></button> <input type="number" disabled="disabled" value="1" class="h-stepper__input"> <button class="h-stepper__plus h-stepper__plus--disabled"></button></div>
  </div>
</div>
`;
